backend/config/settings.py

```
@@ -207,21 +207,23 @@
     "replica": dj_database_url.config(
         env="REPLICA_DATABASE_URL",
         default=os.getenv("DATABASE_URL")
-        or f"sqlite:///{BASE_DIR / 'db.sqlite3'}",  # Falls back to primary in production if replica env is unset
-        conn_max_age=600,
+        or f"sqlite:///{BASE_DIR / 'db.sqlite3'}",
+        conn_max_age=int(os.getenv("CONN_MAX_AGE", "0")),
         conn_health_checks=True,
     ),
 }
 
 for db_name, db_config in DATABASES.items():
```

Ruff (0.15.20)

[warning] Loop control variable `db_name` not used within loop body

Rename unused `db_name` to `_db_name`

(B007)